duplicity-backup.sh

```diff
@@ -361,7 +361,7 @@ mailcmd_nail() {
   ${MAILCMD} -s "${EMAIL_SUBJECT}" -r "${EMAIL_FROM}" "${EMAIL_TO}" < "${LOGFILE}"
 }
 mailcmd_else() {
-  ${MAILCMD} "${EMAIL_SUBJECT}" "${EMAIL_FROM}" "${EMAIL_TO}" < "${LOGFILE}"
+  mailcmd_sendmail
```

Comment:
I merged with dev and committed again, but it's not reflected here in the pull. Should I open a new pull?